### ISSUE TITLE
Abnormal muscle morphology

### ISSUE THREAD

**Issue by jlaw9** [obophenotype/human-phenotype-ontology#4914]
**HPO term**
Abnormal muscle morphology HP:0011805
Abnormality of skeletal muscles HP:0040290

**Suggested revision of textual definition**
Abnormal muscle morphology seems to have the description for Abnormality of skeletal muscles

**Comment by drseb** [comment]
The primary label of HP:0011805 is **Abnormal skeletal muscle morphology**, so I think the current definition is correct. It reads 

> A structural abnormality of a skeletal muscle.

Can you give another description of the issue you are having here?

**Comment by jlaw9** [comment]
For the term `Abnormality of skeletal muscles` ([HP:0040290](https://hpo.jax.org/app/browse/term/HP:0040290)) I don't see a description, yet for `Abnormal muscle morphology` ([HP:0011805](https://hpo.jax.org/app/browse/term/HP:0011805)) I see the description "A structural abnormality of a skeletal muscle."

I thought maybe the current description for HP:0011805 belonged to HP:0040290. If there's no problem, you can go ahead and close the issue.
